extensions/amp-access/0.1/access-expr-impl.jison

```diff
@@ -0,0 +1,130 @@
+
+/* Lexical grammar */
+%lex
+%%
+\s+                       /* skip whitespace */
+"AND"                     return 'AND'
+"OR"                      return 'OR'
+"NOT"                     return 'NOT'
+"NULL"                    return 'NULL'
+"TRUE"                    return 'TRUE'
+"true"                    return 'TRUE'
+"FALSE"                   return 'FALSE'
+"false"                   return 'FALSE'
+"("                       return '('
+")"                       return ')'
+"|"                       return '|'
+"<="                      return 'LTE'
+"<"                       return 'LT'
+">="                      return 'GTE'
+">"                       return 'GT'
+"!="                      return 'NEQ'
+"=="                      return 'EQ'
+"="                       return 'EQ'
+[0-9]+("."[0-9]+)?\b      return 'NUMERIC'
+[a-zA-Z0-9]+              return 'NAME'
+\'[^\']*\'                return 'STRING'
+\"[^\"]*\"                return 'STRING'
+.                         return 'INVALID'
+<<EOF>>                   return 'EOF'
+/lex
+
+
+/* Symbolic tokens */
+
+%token NAME
+%token STRING
+%token NUMERIC
+%token TRUE FALSE
+%token NULL
+%token EOF
+
+
+/* Operators */
+
+%left OR
+%left AND
+%left NOT
+%left EQ
+%left NEQ
+
+
+%%
+
+/* Rules */
+
+result:
+    search_condition EOF
+      {return $1;}
+  ;
+
+search_condition:
```

Comment:
Please add examples (either to the productions or just one big one) of the type of string parsed by this grammar. I think that always really helps with understanding a grammar.